### ISSUE TITLE
Add Missing Methods for Workflow

### ISSUE THREAD

**Issue by mr-palindrome** [novuhq/novu-python#92]
None of the workflow [endpoints](https://api.novu.co/api#/Workflows) are implemented in the python-SDK.

I checked and variefied that none of the workflow APIs are implemented in python-SDK. The examples mentioned in the [Novu documentations](https://docs.novu.co/api-reference/workflows/get-workflows) seems to be wrong. 

Need verification from the maintainers on this. As the `novu/constants.py` (https://github.com/novuhq/novu-python/blob/main/novu/constants.py) doesn't has any `/workflow/` endpoint constant.

<img width="1582" alt="image" src="https://github.com/novuhq/novu-python/assets/56421861/a9e94782-1f86-49a5-9b35-32a9711af611">

<img width="675" alt="image" src="https://github.com/novuhq/novu-python/assets/56421861/08db5ea9-ed3e-4ca0-9a47-fd8cf8366e49">

**Comment by mr-palindrome** [comment]
@Cliftonz @unicodeveloper  can you guys please have a look on this!

**Comment by mr-palindrome** [comment]
> @mr-palindrome We recently renamed Notification Templates to workflows.
> 
> Yes we need to create workflows endpoints (methods) as well end mark notification templates endpoints (methods) as deprecated

So can I work on this and start creating the workflow methods!

**Comment by sumitsaurabh927** [comment]
@mr-palindrome any updates?

**Comment by mr-palindrome** [comment]
> @mr-palindrome any updates?

Yeah the DTO for workflow methods has been completed.. need to add the methods and the test cases. I'll make an initial commit in the respective forked repo.
